### PR TITLE
Add conduwuit and repo for jSynapse to servers metadata

### PR DESCRIPTION
**Pull request by clokep**
Conduwuit is a hard-fork of conduit, so is worthwhile listing as a separate homeserver. Adds the repository for jSynapse, which was missing.

### CODE REVIEW

**Comment by clokep** [comment]
CC @girlbossceo -- not sure if I portrayed conduwuit correctly?

**Comment by girlbossceo** [comment]
Looks good! Is there room for a name change in the future? The name conduwuit occasionally causes confusion as to whether we're a soft fork or hard fork or not, and I would like to eventually get it rebranded (still with the uwu-ness, but detaching from Conduit), just not right now.

**Comment by clokep** [comment]
> Looks good! Is there room for a name change in the future?

I don't see why not! We will just rename it (and maybe leave a reference for formerly known as?)

**Review by girlbossceo — APPROVED** [review]
Sounds good, as the conduwuit maintainer this PR has my approval.

**Review by thibaultamartin — APPROVED** [review]

**Pull request merged**
